stories/features/newTab/components/stats/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+
+import {
+  StyledStatsItemContainer,
+  StyledStatsItem,
+  StyledStatsItemCounter,
+  StyledStatsItemText,
+  StyledStatsItemDescription
+} from './style'
+
+export interface StatsProps {
+  testId?: string
+  children?: React.ReactNode
```

Comment:
I don't think you need to specify `children` as a prop in the Type if it's optional since it's included in the props automatically and by default is optional. I could be wrong, but I think if you remove this it will compile.